src/olympia/stats/views.py

```diff
@@ -149,12 +120,15 @@ def overview_series(request, addon, group, start, end, format):
     start_date, end_date = date_range
     check_stats_permission(request, addon)
 
-    downloads = get_download_series(
-        addon=addon, start_date=start_date, end_date=end_date
-    )
-    updates = get_updates_series(addon=addon, start_date=start_date, end_date=end_date)
-
-    series = zip_overview(downloads, updates)
+    series = []
+    if not switch_is_active('disable-bigquery'):
+        downloads = get_download_series(
+            addon=addon, start_date=start_date, end_date=end_date
+        )
+        updates = get_updates_series(
+            addon=addon, start_date=start_date, end_date=end_date
+        )
+        series = zip_overview(downloads, updates)
 
```

Comment:
Here and below I think we need to indicate that there was a temporary issue in the response as well. Probably return a 503 response somehow.

Reply:
Do you think 503 is accurate? I am wondering how that will work with the js :p It's certainly better that returning empty data so I'll see what I can do